scss/navbar.scss

```
@@ -1,27 +1,27 @@
 $splash-light-purple: #7c408a;
 
-.navbar {
+.navbar:not(.md-layout-nav) {
   position: absolute !important;
   width: 100%;
   z-index: 100;
   padding-top: 1.25em !important;
   top: 0;
-}
 
-.navbar-nav {
-  font-size: 1.25em !important;
-}
+  .navbar-nav {
+    font-size: 1.25em !important;
+  }
 
-.nav-link {
-  padding: 0.5em 1em !important;
-  transition: color 0.25s ease;
-}
+  .nav-link {
+    padding: 0.5em 1em !important;
```

```suggestion
    padding: 10px 15px !important;
```

Match the nav link spacing exactly